### PR TITLE
[Improvement] Allow tcp:// and ssl:// protocols in endpoints for members

### PR DESCRIPTION
**Pull request by nikita-vanyasin**

### CODE REVIEW

File: pkg/exporter/monitor.go [hunk 1]
```diff
@@ -134,7 +135,7 @@ func (m monitor) GetMemberStatus(id driver.ServerID, member driver.ServerHealth)
 		return result, err
 	}
 
-	req.URL, err = setPath(member.Endpoint, shared.ArangoExporterStatusEndpoint)
```
Comment (ajanikow) [inline pkg/exporter/monitor.go:137]: Change Please also `logger.Err(err).Error("GetMemberStatus error")` to Info level

**Review by ajanikow — CHANGES_REQUESTED** [review]
Comments

File: pkg/exporter/monitor.go (hunk 1 above)
Comment (nikita-vanyasin) [reply pkg/exporter/monitor.go:137]: Done. Also dropped one unused file related to endpoints handling

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,7 @@
 # Change Log
 
+- (Improvement) Allow tcp:// and ssl:// protocols in endpoints for members
```
[the remark this answers is not in the document]
Comment (nikita-vanyasin) [reply CHANGELOG.md]: done

**Review by jwierzbo — APPROVED** [review]

**Review by ajanikow — APPROVED** [review]

**Pull request merged**